cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDatabaseBase.java

```diff
@@ -348,6 +348,41 @@ public Response saveAttachment(InputStream in, String name, String contentType,
     }
 
     /**
+     * Removes an attachment from a document.
+     * <p>The object must have the correct <code>_id</code> and <code>_rev</code> and <code>attachmentName</code> values.
+     *
+     * @param object The document to remove attachment as object.
+     * @param attachmentName The attachment name to remove.
+     * @return {@link Response}
+     * @throws NoDocumentException If the document is not found in the database.
```

Comment:
I think `DocumentConflictException` should be in the throws list for the documentation.